src/adapter/graphql/src/scalars/odf_generated.rs

```diff
@@ -284,6 +340,12 @@ pub struct EventTimeSourceFromMetadata {
     pub _dummy: Option<String>,
 }
 
+impl From<odf::EventTimeSourceFromMetadata> for EventTimeSourceFromMetadata {
+    fn from(v: odf::EventTimeSourceFromMetadata) -> Self {
+        Self { _dummy: None }
```

Comment:
These diff looks confusing, but it's just a slight simplification of the codegen (both for GQL, and plain DTO types) that I did for enum variants that don't have any fields. Previously they were treated separately for no good reason.